doc/Customization-guide/Endpoint-SDKs/C-SDK/Porting-guide/index.md

```diff
@@ -4,3 +4,69 @@ title: Porting guide
 permalink: /:path/
 sort_idx: 40
 ---
+
+{% include variables.md %}
+
+* TOC
+{:toc}
+
+
+# Prerequisites
+
+Kaa C SDK can be ported to most platforms with OS, without any OS and it's "endian safe".
```

Comment:
This statement is too abstract and clarified in the next statement below. Merge them to get one paragraph describing platform requirements for C SDK